util/keyfile/src/config.rs

```diff
@@ -14,7 +14,7 @@ pub struct Config {
     pub fog_report_url: Option<String>,
 
     /// Fog Report ID
-    #[clap(long, env = "MC_FOG_REPORT_ID")]
+    #[clap(long, env = "MC_FOG_REPORT_ID", default_value = "")]
```

Comment:
This a default value this will never be `None`, so having it be an `Option<String>` is moot. I think it can just be `String`, and I believe that automatically defaults to empty when `required` is not specified in the `clap` attribute.